workspaces/marketplace/plugins/marketplace/src/plugin.ts

```diff
@@ -91,11 +107,12 @@ export const MarketplaceTabbedPageRouter = marketplacePlugin.provide(
 );
 
 /**
+ * Main marketplace plugin router with tabs and sub-routes.
  * @public
  */
 export const DynamicMarketplacePluginRouter = marketplacePlugin.provide(
   createRoutableExtension({
-    name: 'DynamicMarketplacePluginRouter',
+    name: 'MarketplaceRouter',
```

Comment:
Why changing this name? As long as the export is called DynamicMarketplacePluginRouter and the component, I would keep the old name.